rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/domain/model/LongRunningAction.java

```diff
@@ -725,24 +725,33 @@ protected LRAStatus toLRAStatus(int atomicActionStatus) {
     }
 
     public LRAParticipantRecord enlistParticipant(URI coordinatorUrl, String participantUrl, String recoveryUrlBase,
-                                                  long timeLimit, String compensatorData) throws UnsupportedEncodingException {
-        LRAParticipantRecord participant = findLRAParticipant(participantUrl, false);
-
-        if (participant != null) {
-            participant.setCompensatorData(compensatorData);
-            return participant; // must have already been enlisted
+            long timeLimit, String compensatorData) throws UnsupportedEncodingException {
+        ReentrantLock lock = tryLockTransaction();
+        if (lock == null) {
+            LRALogger.i18nLogger.error_enlistment();
```

Comment:
I [said earlier](https://github.com/jbosstm/narayana/pull/2293#discussion_r1750047056) that:

> we used ERROR for reporting the problem back to the client but WARN for logging inside the coordinator since it is an error for the client but not for the coordinator.

and you gave my comment the "thumbs up" but this log message is logged at the coordinator using `@LogMessage(level = ERROR)` instead of `level = WARN`.

Reply:
Sorry, my mistake. I'll update the PR.